spring-kafka/src/main/java/org/springframework/kafka/core/KafkaOperations.java

```diff
@@ -64,6 +64,16 @@
 	ListenableFuture<SendResult<K, V>> sendDefault(int partition, K key, V data);
 
 	/**
+	 * Send the data to the default topic with the provided key and partition.
+	 * @param partition the partition.
+	 * @param timestamp the timestamp of the record.
+	 * @param key the key.
+	 * @param data the data.
+	 * @return a Future for the {@link SendResult}.
+	 */
+	ListenableFuture<SendResult<K, V>> sendDefault(int partition, Long timestamp, K key, V data);
```

Comment:
OK. We will backport only `timestamp` addition. The `partition` part should stay the same.